README.md

```diff
@@ -1 +1,148 @@
+
+## Features
+
+* `separations.scss` file contains all the margin and padding class generators for your own stylesheet.
+* `layout.scss` contains a series of classes that can be used to define a page's layout using flexbox.
+
+## Installation
+
+1.  intall the package with `npm` or `yarn`
+
+#### npm
+
+```bash
+npm i wolox-layout
+```
+
+#### yarn
+
+```bash
+yarn add wolox-layout
+```
+
+2- Include the wolox-bootstrap.scss before the index file of your CSS/SCSS to have paddings, margins, and layout already generated all toghether. Or you can import them separately.
+
+If you want to import the 'wolox-bootstrap' in this way:
+
+```scss
+@import '~wolox-bootstrap/wolox-bootstrap';
+ // or
+@import '~wolox-bootstrap/layout';
+@import '~wolox-bootstrap/separations';
+```
+
+## Usage
+
+### General import of Wolox-boostrap
+
+This will generate a series of classes like:
+```sass
+.m-0 { margin: 0; }
+.m-1 { margin: 10px; }
+...
+.m-20 { margin: 200px; }
+
+.m-top-0 { margin: 0; }
+.m-top-1 { margin: 10px; }
+...
+.m-top-20 { margin: 200px; }
+
+@media (max-width: 960px) {
+.p-top-0-sm { padding: 0; }
+.p-top-1-sm { padding: 10px; }
+...
+.p-top-20-sm { padding: 200px; }
+}
+
+// And so on for every type of margin and padding from 0 to 200px.
+```
+### Layout API interface
+To generate a flexbox layout you can do:
+```html
+<div class="column middle">...</div>
+
+<!-- this is the equivalent to: -->
+```
+```scss
+.my-custom-class {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+```
+
+In summary, for vertical alignment we use `top`, `middle`, `bottom` and for horizontal alignment: `start`, `center`, `end`.
+
+| Column  | Class name  |
+| :------------ |:---------------:|
+| `align-items: center`      | `column center` |
+| `align-items: flex-end`      | `column end` |
+| `align-items: flex-start`      | `column start` |
+| `justify-content: center`      | `column middle` |
+| `justify-content: flex-end`      | `column bottom` |
+| `justify-content: flex-start`      | `column top` |
+| `justify-content: space-between`      | `column space-between` |
+| `justify-content: space-around`      | `column space-around` |
+
+
+| Row  | Class name  |
+| :------------ |:---------------:|
+| `align-items: center`      | `row middle` |
+| `align-items: flex-end`      | `row bottom` |
+| `align-items: flex-start`      | `row top` |
+| `justify-content: center`      | `row center` |
+| `justify-content: flex-end`      | `row end` |
+| `justify-content: flex-start`      | `row start` |
+| `justify-content: space-between`      | `row space-between` |
+| `justify-content: space-around`      | `row space-around` |
+
+For responsive classes (with 960px as breakpoint) append `-sm` suffix to the class.
+
+
+```html
+<div class="row space-between space-around-sm">
+  ...
+</div>
+```
+### Margins and Paddings generator
+In case of including only the separation file the way to create this custom classes is
```

Comment:
This looks like a cool feature but I think a simpler way to achieve this is by using the default variables in the link before